trailblazer/store/crud/update.py

```diff
@@ -233,12 +233,32 @@ def update_analysis_comment(self, case_id: str, comment: str) -> None:
         session.commit()
         LOG.info(f"Adding comment {comment} to analysis {analysis.case_id}")
 
+    @staticmethod
+    def update_analysis_delivery(
+        analysis: Analysis, is_delivered: bool, user: User | None = None
+    ) -> None:
+        session: Session = get_session()
+        if is_delivered and not analysis.delivery:
```

Comment:
Why do we need the `not analysis.delivery` check?

Reply:
Because it crashes if we try add another entry to the database - they should only be delivered once right?

Reply:
Double checked with @beatrizsavinhas and seems as if this is good for now, but we can think of whether there is a benefit to being able to track multiple deliveries.